scripts/ci/npm/publish-libs.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=${DIR}/../../..
+DIST_DIR=${ROOT_DIR}/dist
+PROJECTS_DIR=${ROOT_DIR}/projects
+
+VERSION_IN_PACKAGE_JSON=`node -p "require('$ROOT_DIR/package.json')".version;`;
+
+if [ "$TRAVIS_BRANCH" = "master" ] ; then
+  NEW_LIBRATY_VARSION=VERSION_IN_PACKAGE_JSON
+else
+  NEW_LIBRATY_VARSION="${VERSION_IN_PACKAGE_JSON}-${TRAVIS_BUILD_NUMBER}"
+fi
+
+PROJECTS=(
+    'aca-shared'
+    'aca-about'
+    'adf-office-services-ext'
+    'aca-content'
+    'aca-folder-rules'
+    'aca-settings'
+    'aca-testing-shared'
+);
+
+for PROJECT in "${PROJECTS[@]}"
+do
+    echo "Update ${PROJECT} version to ${NEW_LIBRATY_VARSION}"
+
+    cd $PROJECTS_DIR/${PROJECT}
+    npm version ${NEW_LIBRATY_VARSION};
+done
+
+echo -e "\n\nBuild projects"
+cd ${ROOT_DIR}
+npm run build-libs
+
+for PROJECT in "${PROJECTS[@]}"
+do
+    echo "Publish ${PROJECT}"
+
+    cd $DIST_DIR/${PROJECT}
+
+    echo 'strict-ssl=false' >> .npmrc
+    echo 'registry=http://${NPM_REGISTRY_ADDRESS}' >> .npmrc
+    echo '//${NPM_REGISTRY_ADDRESS}/:_authToken="${NPM_REGISTRY_TOKEN}"' >> .npmrc
+
+    # npm publish
```

Comment:
commented from obvious reason :)

Reply:
can we have a dry run option ?

Reply:
Added variable in travis - `PUBLISH_PROJECTS`, whenever it is going to be set to false, we wont publish npm packages